### ISSUE TITLE
Add additional AHI RGB composites

### ISSUE THREAD

**Issue by djhoese** [ssec/polar2grid#258]
This has been discussed in recent Geo2Grid meetings for the next 1.0.1 patch release. We have a lot of possible RGBs for ABI, but never included them as part of the AHI available composites. I think this was mainly due to limited time at the time of G2G 1.0.0's release. This issue is to document my brainstorming for adding any additional RGBs and document any reasons why some of them might not be possible.

Note: This may not include adding these composites to the AHI HRIT reader given the limited number of channels available in that format.

## ABI but not AHI

Composites that exist for ABI that don't exist for AHI in G2G:

- [ ] airmass
- [ ] ash
- [ ] dust
- [ ] fog
- [ ] night_microphysics

**Comment by djhoese** [comment]
## Airmass

https://weather.msfc.nasa.gov/sport/training/quickGuides/rgb/QuickGuide_AirmassRGB_NASA_SPoRT.pdf

The EUMETSAT Airmass RGB recipe has been defined in Satpy as using the 10.8um channel. AHI only has a 10.4um channel so this should be fine as a substitute.

## Ash

Same as airmass for the 10.8um versus 10.4um, but the "generic" Satpy recipe also wants a 12.0um but AHI has a 12.4um.

## Dust

10.8um versus 10.4um

## Fog

10.8um versus 10.4um and 12.0um versus 12.4um

## Night Microphysics

10.8um versus 10.4um and 12.0um versus 12.4um

So given these differences, I think it should be fine to re-define the recipes for AHI to use the channels with the closest wavelength.

**Comment by djhoese** [comment]
Little bit of a hiccup: ABI's recipes are working "by accident" because C14's wavelength ranges from 10.8um to 11.60um (nominal/central is 11.2um). So when the generic recipe asks for 10.8um it is finding C14 (11.2um). But C13 is (10.10um, 10.35um, 10.60um) so not necessarily worse.

For AHI, B14 doesn't go all the way to 10.8um but instead (11.0, 11.2, 11.4), so should B13 be used (10.2, 10.4, 10.6)?

**Comment by djhoese** [comment]
I talked to @scottlindstrom and he said for Ash, Dust, and Fog RGBs (all the same band combinations) the 10.8um/8.6um BT difference should use 11.2um - 8.6um because 11.2um better discriminates cloud properties in comparison with 8.6um versus doing the difference with 10.4um.

I then got a screenshot of a slide from @mraspaud from Akihiro Shimizu of JMA (Japan Meteorological Agency). Their slide agrees with what Scott said with one other change. They have the night_microphysics 10.8um - 3.9um be 11.2um - 3.9um. Scott says he'd probably still use 10.4um for this, but that it won't make a big difference. I'll go with the JMA employee so we have some justification for it later.
